browser/plugins/test/network-breadcrumbs.test.js

```diff
@@ -10,25 +10,32 @@ const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
 const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
 const CROSS_ORIGIN_ENDPOINT_WITHOUT_CORS_HEADERS = `http://${isSafari ? '0.0.0.0' : 'localhost'}:55854/?nocors`
 
-let client
-
-beforeEach(() => {
-  // setup client and plugin
-  client = new Client(VALID_NOTIFIER)
-  client.configure({
-    apiKey: 'aaaa-aaaa-aaaa-aaaa',
-    endpoint: '/echo/reports',
-    sessionEndpoint: '/echo/sessions'
+describe('plugin: network breadcrumbs', () => {
+  let client
+
+  beforeEach(() => {
+    // setup client and plugin
+    client = new Client(VALID_NOTIFIER)
+    try {
+      client.configure({
+        apiKey: 'aaaa-aaaa-aaaa-aaaa',
+        endpoints: {
+          notify: '/echo/reports',
+          sessions: '/echo/sessions'
+        }
+      })
```

Comment:
Have we got an automated test to ensure that the deprecated API works as expected? I scanned but couldn't find anything (although that might reflect more on my search skills).

Reply:
Not automated, no. This is what I went through with Alex/Dave manually. I'll get it set up on maze though.